### PR TITLE
Week1 - Mohammed Saif ( Create database and tables + Select queries) 

### PR DESCRIPTION
**Pull request by MoSaif00**
In this homework I had to do 2 exercises : 
1- Create a database, tables, and insert rows to the tables.
2- use the select query to answer many questions from world.sql database

### CODE REVIEW

File: Week1/homework/2-selectQueries.js [hunk 1]
```diff
@@ -3,15 +3,48 @@ const connection = mysql.createConnection({
...
 connection.connect(function (err) {
   if (err) throw err;
   console.log('Node.js connected to MySQl!');
 });
-// use the = database
-connection.query('use world', function (err, result) {
-  if (err) throw err;
-  console.log('we are in the database');
-});
+
```
Comment (Breus) [inline Week1/homework/2-selectQueries.js:12]: All good, well done!

**Comment by Breus** [comment]
Well done on the homework. Everything seems to work as intended.

**Review by MoSaif00 — COMMENTED** [review]
> Well done on the homework. Everything seems to work as intended.

Thank you very much @Breus for reviewing my homework, I have made some changes according to your suggestions. 
I also learned about using " if not exist". I actually was not aware of it, I did look it up because when I was executing much time I had to delete the previous database or comment on the codes that I already executed. 
Thank you again for your comments and feedback.

File: Week1/connection-test.js [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
-  host     : 'localhost',
-  user     : 'hyfuser',
-  password : 'hyfpassword',
-  database : 'userdb',
-  
-  // port : xxxx // Uncomment this line and replace xxxx with the selected port number if you are not using default 3306. I also suggest to download MySQL version 5.7 because recent versions has authentication problems
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'userdb',
```
[the remark this answers is not in the document]
Comment (Breus) [reply Week1/connection-test.js:6]: Got it.

**Pull request closed without merging**